pkg/cmd/server/api/types.go

```diff
@@ -292,6 +292,16 @@ type MasterConfig struct {
 	// JenkinsPipelineConfig holds information about the default Jenkins template
 	// used for JenkinsPipeline build strategy.
 	JenkinsPipelineConfig JenkinsPipelineConfig
+
+	// AuditConfig holds information related to auditing capabilities.
+	AuditConfig AuditConfig
+}
+
+// AuditConfig holds configuration for the audit capabilities
+type AuditConfig struct {
+	// If this flag is set, basic audit log will be printed in the logs.
+	// The logs contains, method, user and a requested URL.
+	BasicAuditEnabled *bool
```

Comment:
Let's just call this `Enabled`.  In the future we can add handler configs for the destinations (basic, files, events, etc) and keep this master flag for the on/off.